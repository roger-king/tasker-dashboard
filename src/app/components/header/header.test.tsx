import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Header from './index';

Enzyme.configure({ adapter: new Adapter() });

describe('<Header />', () => {
    it('Matches the snapshot', () => {
        const header = renderer
            .create(
                <MemoryRouter>
                    <Header gridArea="header" />
                </MemoryRouter>,
            )
            .toJSON();
        expect(header).toMatchSnapshot();
    });
});
