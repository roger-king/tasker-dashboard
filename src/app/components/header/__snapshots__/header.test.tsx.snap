// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Matches the snapshot 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 eeRfnU"
>
  <div
    className="StyledBox-sc-13pk1d4-0 jEyert"
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <img
      className="StyledImage-ey4zx9-0 fmkmCl"
      src="/images/flash.png"
      width="40px"
    />
    <h2
      className="StyledHeading-sc-1rdh4aw-0 yhirM"
    >
      TASKER.
    </h2>
  </div>
  <div
    className="StyledBox__StyledBoxGap-sc-13pk1d4-1 gufrXq"
  />
  <div
    className="StyledBox-sc-13pk1d4-0 kqaBOm"
  >
    <div
      className="StyledBox-sc-13pk1d4-0 ekjAwv"
    >
      <button
        className="StyledButton-sc-323bzc-0 jfbKsv"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        type="button"
      >
        Overview
      </button>
      <div
        className="StyledBox__StyledBoxGap-sc-13pk1d4-1 hyyRSo"
      />
      <button
        className="StyledButton-sc-323bzc-0 jfbKsv"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        type="button"
      >
        Tasks
      </button>
    </div>
    <div
      className="StyledBox-sc-13pk1d4-0 eBbbPL"
    >
      <button
        className="StyledButton-sc-323bzc-0 lgwBfH"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        type="button"
      >
        <svg
          aria-label="Configure"
          className="StyledIcon-ofa7kd-0 jAaLrm"
          viewBox="0 0 24 24"
        >
          <path
            d="M16,15 C20.0089021,14.9354541 23,11.9673591 23,8 C23,4.98813056 22.0029673,5.9851632 21,7 C20.0089021,7.97922849 18,10 18,10 L14,6 C14,6 16.0207715,3.99109792 17,3 C18.0148368,1.99703264 18.0148368,1 16,1 C12.0326409,0.999999999 9.05307486,3.99109792 9,8 C9.04154304,8.97626113 9,11 9,11 C7.11486635,12.8970031 4.65923194,15.3526375 3,17 C0.0682492584,19.9436202 4.05637975,23.9317507 7,21 C8.65052042,19.3376102 11.1126942,16.8754364 13,15 C13,15 15.0237389,14.958457 16,15 Z"
            fill="none"
            stroke="#000"
            strokeWidth="2"
          />
        </svg>
      </button>
      <button
        aria-label="Open Drop"
        className="StyledButton-sc-323bzc-0 lgwBfH"
        id="header-notification-btn"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        type="button"
      >
        <svg
          aria-label="Notification"
          className="StyledIcon-ofa7kd-0 jAaLrm"
          viewBox="0 0 24 24"
        >
          <path
            d="M4,19 L4,9 C4,4.582 7.582,1 12,1 C16.418,1 20,4.582 20,9 L20,19 M1,19 L23,19 M15,19 L15,20 C15,21.657 13.657,23 12,23 C10.343,23 9,21.657 9,20 L9,19"
            fill="none"
            stroke="#000"
            strokeWidth="2"
          />
        </svg>
      </button>
      <button
        aria-label="Open Drop"
        className="StyledButton-sc-323bzc-0 lgwBfH"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        type="button"
      >
        <svg
          aria-label="User"
          className="StyledIcon-ofa7kd-0 jAaLrm"
          viewBox="0 0 24 24"
        >
          <path
            d="M8,24 L8,19 M16,24 L16,19 M3,24 L3,19 C3,14.0294373 7.02943725,11 12,11 C16.9705627,11 21,14.0294373 21,19 L21,24 M12,11 C14.7614237,11 17,8.76142375 17,6 C17,3.23857625 14.7614237,1 12,1 C9.23857625,1 7,3.23857625 7,6 C7,8.76142375 9.23857625,11 12,11 Z"
            fill="none"
            stroke="#000"
            strokeWidth="2"
          />
        </svg>
      </button>
    </div>
  </div>
</div>
`;
